vendor/engines/sanger_sequencing/app/views/sanger_sequencing/sanger_products/edit.html.haml

```diff
@@ -0,0 +1,22 @@
+= content_for :h1 do
+  = current_facility
+= content_for :sidebar do
+  = render "admin/shared/sidenav_product", sidenav_tab: @product.class.name.pluralize.underscore
+= content_for :tabnav do
+  = render "admin/shared/tabnav_product", secondary_tab: "sanger"
+
+%h2= text("title")
```

Comment:
This part is also in show and I think it's the same. WDYT about extracting it to a partial?

Reply:
I created a partial for product manage (the first 6 lines that it's used in several places)